schema.go

```diff
@@ -43,7 +44,7 @@ func (s *defaultSchema) register(n *node) {
 
 // find finds provideFunc by its reflect.Type and Tags.
 func (s *defaultSchema) find(t reflect.Type, tags Tags) (*node, error) {
-	nodes, ok := s.nodes[t]
+	nodes, ok := s.list(t)
```

Comment:
Need some like that for group resolve, I think

Reply:
Yeah.  I should add a test case for group resolve across multiple containers.

Reply:
Ok added it to the group method too and included test case to show it working with groups.